src/server/redis_reply.h

```diff
@@ -40,6 +40,15 @@ std::string Integer(T data) {
   return ":" + std::to_string(data) + CRLF;
 }
 
+template <typename T, std::enable_if_t<std::is_integral_v<T>, int> = 0>
+std::string MultiInteger(const std::vector<T> &multi_data) {
```

Comment:
A quick question: would other Redis response can be help with `MultiInteger`?

Reply:
(Oh and @PragmaTwice and can we support a `vector<bool>` version to convert `vector<bool>` to 0/1?)

Reply:
not sure since resp3 supports boolean.

Reply:
It's seems only here and `SMIsMember` can be help with `MultiInteger`. Should we give up `MultiInteger`?

Reply:
You can put it in `src/commands/cmd_bloom_filter.cc` in anonymous namespace as a helper first. If you find any other user would use it, you can move it out.

Reply:
this function is ok to me.